genesis/src/main.rs

```diff
@@ -305,26 +310,17 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         )
         .get_matches();
 
-    let bootstrap_leader_pubkey_file = matches.value_of("bootstrap_leader_pubkey_file").unwrap();
-    let bootstrap_vote_pubkey_file = matches.value_of("bootstrap_vote_pubkey_file").unwrap();
-    let bootstrap_stake_pubkey_file = matches.value_of("bootstrap_stake_pubkey_file").unwrap();
-    let bootstrap_storage_pubkey_file = matches.value_of("bootstrap_storage_pubkey_file");
-    let faucet_pubkey_file = matches.value_of("faucet_pubkey_file");
     let faucet_lamports = value_t!(matches, "faucet_lamports", u64);
     let ledger_path = PathBuf::from(matches.value_of("ledger_path").unwrap());
     let bootstrap_leader_lamports = value_t_or_exit!(matches, "bootstrap_leader_lamports", u64);
     let bootstrap_leader_stake_lamports =
         value_t_or_exit!(matches, "bootstrap_leader_stake_lamports", u64);
 
-    let bootstrap_leader_pubkey = pubkey_from_file(bootstrap_leader_pubkey_file)?;
-    let bootstrap_vote_pubkey = pubkey_from_file(bootstrap_vote_pubkey_file)?;
-    let bootstrap_stake_pubkey = pubkey_from_file(bootstrap_stake_pubkey_file)?;
-    let bootstrap_storage_pubkey =
-        if let Some(bootstrap_storage_pubkey_file) = bootstrap_storage_pubkey_file {
-            Some(pubkey_from_file(bootstrap_storage_pubkey_file)?)
-        } else {
-            None
-        };
+    let bootstrap_leader_pubkey = required_pubkey(&matches, "bootstrap_leader_pubkey_file")?;
+    let bootstrap_vote_pubkey = required_pubkey(&matches, "bootstrap_vote_pubkey_file")?;
+    let bootstrap_stake_pubkey = required_pubkey(&matches, "bootstrap_stake_pubkey_file")?;
+    let bootstrap_storage_pubkey = pubkey_of(&matches, "bootstrap_storage_pubkey_file");
+    let faucet_pubkey = pubkey_of(&matches, "faucet_pubkey_file");
```

Comment:
Hehe, now the pubkey params are lined up beatifically regardless of mandatory or optional.